src/main/java/org/opentripplanner/transit/model/site/Station.java

```diff
@@ -142,6 +155,10 @@ public ZoneId getTimezone() {
     return timezone;
   }
 
+  public Accessibility getWheelchairAccessibility() {
```

Comment:
The only way to interperet "wheelchair-accessible" on a station is that it is the default value for the stops, and then the stops should be able to override it. If so, "the only implication is that the station centroid is reachable by wheelchair" is wrong. Because, we should check ways and stops but not the free-edge to the centroid when doing wheelchair routing. Example: Default for Station is NOT_ASSESSABLE, then we would not be able to reach a ACCESSABLE stop in that station, just because we continue beyond the "passenger" destination to prioritise the path. Note! We reach the destination, before the search is terminated. If we can not traverse the last edge(to the centroid) we fail to find a valid result.

Reply:
As per the discussion in the developer meeting I removed the accessibility info from the Stations again.